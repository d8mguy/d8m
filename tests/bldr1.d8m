// test of strings pkg
val strgpkgTest = \imp(xx : integer, yy : float) {
    import go "strings"
    var bldr : strings.Builder
    bldr.WriteString("xxx yyy zzz")
    fmt.Fprintf(bldr, "; %6.3f; %d; ", yy, xx)
    bldr.WriteString("aaa bbb ccc")
    bldr.String()
}
strgpkgTest(3498, 55.37402)
