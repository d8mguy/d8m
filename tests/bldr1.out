xxx yyy zzz; 55.374; 3498; aaa bbb ccc
